quickstarts/EAP7-1916-add-ee-security-to-EAP-quickstarts.adoc

```diff
@@ -0,0 +1,108 @@
+= Add ee-security to the EAP quickstarts
+:author:            Farah Juma
+:email:             fjuma@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The ee-security quickstart is currently a community-only quickstart because it's not part of the EAP quickstarts repo.
+Nothing else demonstrates how to configure EE security so this should be added to the EAP quickstarts as well.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/EAP7-1916[EAP7-1916]
+
+=== Related Issues
+
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [ ] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+// Remove the x next to the relevant field if the feature in question is not relevant
+// to that kind of WildFly installation
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+
+* [x] Managed domain
+
+* [x] OpenShift s2i
+
+* [x] Bootable jar
+
+== Requirements
+
+=== Hard Requirements
```

Comment:
Is update to Jakarta APIs in scope of this RFE?

Reply:
@OndrejKotek Confirmed with @emmartins today that updates to Jakarta namespace will be handled as part of a separate RFE.